test/fixtures/config.rb

```diff
@@ -0,0 +1,84 @@
+  # deprecated code
+  c.treat_deprecation_warnings_as_errors = false
+
+  # Warn on all deprecations. Setting this to `true` will cause the library to
+  # warn every time a piece of deprecated code is called. The `false` (default)
+  # behaviour is to only issue a warning once for each call site in your code.
+  c.warn_on_all_deprecations = false
+
+  # The developer token is required to authenticate that you are allowed to
+  # make API calls.
+  c.developer_token = 'INSERT_DEVELOPER_TOKEN_HERE'
+
+  # Authentication tells the API that you are allowed to make changes to the
+  # specific account you're trying to access.
+  # The default method of authentication is to use a refresh token, client id,
+  # and client secret to generate an access token.
+  c.client_id = 'INSERT_CLIENT_ID_HERE'
+  c.client_secret = 'INSERT_CLIENT_SECRET_HERE'
+  c.refresh_token = 'INSERT_REFRESH_TOKEN_HERE'
+
+  # You can also authenticate using a service account. If "keyfile" is
+  # specified below, then service account authentication will be assumed and
+  # the above authentication fields ignored. Read more about service account
+  # authentication here:
+  # https://developers.google.com/google-ads/api/docs/oauth/service-accounts
+  # c.keyfile = 'path/to/keyfile.json'
+  # c.impersonate = 'INSERT_EMAIL_ADDRESS_TO_IMPERSONATE_HERE'
+
+  # Alternatively, you may specify your own custom authentication, which can be:
+  # A `Google::Auth::Credentials` uses a the properties of its represented
+  # keyfile for authenticating requests made by this client.
+  # A `String` will be treated as the path to the keyfile to be used for the
+  # construction of credentials for this client.
+  # A `Hash` will be treated as the contents of a keyfile to be used for the
+  # construction of credentials for this client.
+  # A `GRPC::Core::Channel` will be used to make calls through.
+  # A `GRPC::Core::ChannelCredentials` for the setting up the RPC client. The
+  # channel credentials should already be composed with a
+  # `GRPC::Core::CallCredentials` object.
+  # A `Proc` will be used as an updater_proc for the Grpc channel. The proc
+  # transforms the metadata for requests, generally, to give OAuth credentials.
+  # To use one of these methods, uncomment the following line and add some code
+  # to look up one of the authentication methods listed above. If set, the
+  # authentication field will override the client_id, client_secret, and
+  # refresh_token fields above.
+  # c.authentication = INSERT_AUTHENTICATION_METHOD_HERE
+
+  # Required for manager accounts only: Specify the login customer ID used to
+  # authenticate API calls. This will be the customer ID of the authenticated
+  # manager account. If you need to use different values for this field, then
+  # make sure fetch a new copy of the service after each time you change the
+  # value.
+  # c.login_customer_id = 'INSERT_LOGIN_CUSTOMER_ID_HERE'
+
```

Comment:
`linked_customer_id` is missing in this config file example

Reply:
Good catch, thanks! Added it in example config.